contrib/util/ccda_import/import_ccda.php

```diff
@@ -123,32 +128,40 @@ function outputMessage($message): void
 use OpenEMR\Common\Uuid\UuidRegistry;
 use OpenEMR\Services\Cda\CdaComponentParseHelpers;
 
+
+// clear the log so we only keep last session. Saves cleanup time.
+unlink($GLOBALS['OE_SITE_DIR'] . "/documents/temp/log.txt");
+
 // show parameters (need to do after globals)
-outputMessage("ccda directory: " . $argv[1]);
-outputMessage("site: " . $_SESSION['site_id']);
-outputMessage("openemr path: " . $openemrPath);
+outputMessage("OpenEMR path: " . $openemrPath);
+outputMessage("CCDA Imports Location: " . $argv[2]);
+outputMessage("Authority: " . $argv[1]);
```

Comment:
Is this backwards compatible with this script used in other places (ie. docker environments). If not, is there a way to do that without needing to update all the dev dockers (for example, putting the new argument at the end or getting it in way that does not rely on ordering of parameters)?